tests/TestCase.py

```diff
@@ -106,6 +108,30 @@ def fetch(self, route, data=None, method=None):
     def mock_start_response(self, *args, **kwargs):
         pass
 
+    def register_mocks(self):
+        """Configure the default mock classes for all services which needs to be mocked.
+        The mocks can now be overriden.
+        A package could in its service provider call a method which could update one of the mock
+        class with the one installed.
+        """
+        # here we will configure all default available mocks
+        self.application.bind("mock.mail", "src.masonite.tests.mocks.MockMail")
```

Comment:
Why do this rather than just bind the mock class directly?

Reply:
I changed this to bind to the container directly in the service provider. This will allow us to have packages that easily binds their mocks to the container